src/query/plan/operator.cpp

```diff
@@ -3203,9 +3231,7 @@ bool Merge::MergeCursor::Pull(Frame &frame, ExecutionContext &context) {
       if (pull_input_) {
         // if we have just now pulled from the input
         // and failed to pull from merge_match, we should create
-        __attribute__((unused)) bool merge_create_pull_result = merge_create_cursor_->Pull(frame, context);
```

Comment:
This was removed because it was protection that merge_create always returns true and if it didn't it would throw an error but now if user doesn't have permissions to create then there is a way that merge_create could return false.